content/en/docs/guides/auth.md

```diff
@@ -213,8 +213,8 @@ and authorization in various languages.
 | Python   | [Python Example]                          | [Python Documentation] |
 
 
-[Go Example]: https://github.com/grpc/grpc-go/tree/master/examples/features/encryption
-[Go Documentation]: https://github.com/grpc/grpc-go/tree/master/examples/features/encryption#encryption
+[Go Example]: https://github.com/grpc/grpc-go/tree/master/examples/features/authentication
+[Go Documentation]: https://github.com/grpc/grpc-go/tree/master/examples/features/authentication#encryption
```

Comment:
This table is for language specific TLS examples. So the Go links seems to correctly point to the encryption directory. The table just below this is for OAuth examples, in which the Go row points to the links you've added in this PR.
https://github.com/grpc/grpc.io/blob/f4a96fb104532683f6fcf68408268550ef0c1ba6/content/en/docs/guides/auth.md?plain=1#L224-L238